components/tokens/texts/textinfo.vue

```diff
@@ -1,13 +1,13 @@
 <template>
-  <p><slot /></p>
+  <p class="inline-block"><slot /></p>
 </template>
 
 <script>
 export default {}
 </script>
 
 <style lang="postcss" scoped>
-p {
+div {
```

Comment:
You forgot to revert this:
```suggestion
p {
```